Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2008-10-15
Initial Package Version: 2.5
Upstream Status: unknown
Origin: Opensuse
Description: Allows agg-2.5 to link with libtool-2.2

diff -Naur agg-2.5.orig/configure.in agg-2.5/configure.in
--- agg-2.5.orig/configure.in	2006-10-09 05:06:36.000000000 +0100
+++ agg-2.5/configure.in	2008-10-15 22:36:39.000000000 +0100
@@ -121,6 +121,13 @@
   AC_MSG_WARN([*** X11 not found! Omitting X11 layer.])
 fi
 AM_CONDITIONAL(ENABLE_X11,[test x$no_x = x -a xno != x$enable_platform -a x$win32_host != xyes])
+if test x"$x_includes" = x; then
+ x_includes="/usr/include"
+fi
+AC_SUBST(x_includes)
+if test x"$x_libraries" = x; then
+ x_libraries="/usr/lib"
+fi
 AC_SUBST(x_includes)
 AC_SUBST(x_libraries)
 dnl ###############################################
